public/locales/fr/common.json

```
@@ -0,0 +1,56 @@
+{
+  "scrobble": "Scrobble",
+  "getScrobbling": "Début du scrobbling",
+  "loading": "Chargement...",
+  "artist": "Artiste",
+  "song": "Musique",
+  "title": "Titre",
+  "album": "Album",
+  "albumArtist": "Artiste de l'album",
+  "history": "Historique",
+  "yourProfile": "Votre profil",
+  "language": "Langage",
+  "translateYourLanguage": "Vous voulez ajouter votre langage?",
+  "logIn": "Se connecter",
+  "logOut": "Se déconnecter",
+  "now": "Maintenant",
+  "custom": "Personnaliser...",
+  "timestamp": "Timestamp",
+  "lastfmWillRejectOldTimestamps": "Last.fm rejecte les scrobbles datant de plus de deux semaines.",
+  "noSongsScrobbled": "Aucune musique n'a été scrobblé !",
+  "songsWillAppearHere": "Les pistes apparaîtront ici une fois que vous les aurez scrobblés.",
+  "clearHistory": "Nettoyer l'historique ",
+  "scrobbleAgain": "Scrobbler encore",
+  "copyToEditor": "Le copier à l'éditeur",
+  "swapArtistTitle": "Échanger l'artiste et le morceau",
```

Would it be possible to make this consistent with the translation for `title` or it wouldn't translate well?
Something like this:
```suggestion
  "swapArtistTitle": "Échanger l'artiste et le titre",
```

---

It's better 😉